### PR TITLE
Fix sql alchemy version

### PR DESCRIPTION
**Pull request by Aravind-psiog**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ pydantic>=1.10.1
 python-socketio>=5.7.1
 requests>=2.24.0
 simplejson>=3.17.6
-sqlalchemy>=1.4.37
+sqlalchemy>=1.4.37,<=1.4.41
```
Comment (wjcunningham7) [inline requirements.txt:15]: @Aravind-psiog is it possible to make the upper limit `<2.0.0` since anything in 1.4.x should contain bugfixes

Comment (Aravind-psiog) [reply requirements.txt]: @wjcunningham7 Changes are done.

File: requirements.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ pydantic>=1.10.1
 python-socketio>=5.7.1
 requests>=2.24.0
 simplejson>=3.17.6
-sqlalchemy>=1.4.37
+sqlalchemy<2.0.0
```
Comment (wjcunningham7) [inline requirements.txt:15]:
```suggestion
sqlalchemy>=1.4.37,<2.0.0
```

**Review by wjcunningham7 — APPROVED** [review]

**Review by wjcunningham7 — APPROVED** [review]

**Comment by Prasy12** [comment]
@wjcunningham7 Looks like the dispatcher tests are failing, can you please take a look when you have time?

**Review by wjcunningham7 — APPROVED** [review]

**Pull request merged**